src/RAI.jl

```diff
@@ -20,10 +20,11 @@ APIs.
 """
 module RAI
 
-import Pkg
+import TOML
 
 # Used for sending the User Agent.
-const PROJECT_VERSION = Pkg.project().version
+project_toml = joinpath(dirname(@__DIR__), "Project.toml")
+const PROJECT_VERSION = TOML.parsefile(project_toml)["version"]
```

Comment:
Previously, the type of `PROJECT_VERSION` was `VersionNumber`, but now it will be a `String`. Somewhat of a breaking change. Is this okay?

Reply:
Yeah, it's an internal only field i think, so it seems fine to me.